lib/onc_certification_g10_test_kit/limited_scope_grant_test.rb

```diff
@@ -39,9 +39,14 @@ def possible_resources
     end
 
     def scope_granting_access?(resource_type, scopes)
-      scopes.any? do |scope|
-        scope.start_with?("patient/#{resource_type}", 'patient/*') && scope.end_with?('*', 'read')
-      end
+      scopes
+        .select { |scope| scope.start_with?("patient/#{resource_type}", 'patient/*') }
+        .any? do |scope|
+          _type, resource_access = scope.split('/')
+          _resource, access_level = resource_access.split('.')
+
+          access_level.match?(/\A(\*|read|c?ru?d?s?\b)/)
+        end
```

Comment:
Our reference server doesn't allow us to limit scopes to specific resources when a `*` type scope is used. I think this is a change that might warrant it's own PR (or an update to the reference server to allow `*` scopes to be expanded and selected.

![Screen Shot 2022-08-19 at 4 23 22 PM](https://user-images.githubusercontent.com/41651655/185701250-7b5390d4-2804-48e2-8676-513901fe9d9a.png)
![Screen Shot 2022-08-19 at 4 21 11 PM](https://user-images.githubusercontent.com/41651655/185701265-8c7723a1-e117-4989-9f6e-286292bef9bd.png)
![Screen Shot 2022-08-19 at 4 23 48 PM](https://user-images.githubusercontent.com/41651655/185701322-5706bbe4-3fa1-444e-ac6c-bbb0fd445c6b.png)

Reply:
I don't think that is a requirement.

Reply:
Ya not a requirement. It's really a reference server issue since we can't pass the limited scope test when '*' resource scopes are requested

Reply:
Yeah, I updated all of the defaults to not use wildcard scopes.